classes/options-general.php

```diff
@@ -720,37 +720,37 @@ class='howto'><?php _e( 'HTML5 - more precise, may prompt users for permission,
 
 									<div class="postbox">
 
-										<h3><?php _e( 'Map Style Defaults', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Map Style Defaults', 'SimpleMap' ); ?></h3>
 
 										<div class="inside">
-											<p class="sub"><?php printf( __( 'To insert SimpleMap into a post or page, type this shortcode in the body: %s', 'simplemap' ), '<code>[simplemap]</code>' ); ?></p>
+											<p class="sub"><?php printf( __( 'To insert SimpleMap into a post or page, type this shortcode in the body: %s', 'SimpleMap' ), '<code>[simplemap]</code>' ); ?></p>
 
 											<div class="table">
 												<table class="form-table">
 
 													<tr valign="top">
 														<td width="150"><label
-																for="map_width"><?php _e( 'Map Size', 'simplemap' ); ?></label>
+																for="map_width"><?php _e( 'Map Size', 'SimpleMap' ); ?></label>
 														</td>
 														<td>
 															<label for="map_width"
-															       style="display: inline-block; width: 60px;"><?php _e( 'Width:', 'simplemap' ); ?> </label>
+															       style="display: inline-block; width: 60px;"><?php _e( 'Width:', 'SimpleMap' ); ?> </label>
 															<input type="text" name="map_width" id="map_width" size="13"
 															       value="<?php echo esc_attr( $map_width ); ?>"/><br/>
 															<label for="map_height"
-															       style="display: inline-block; width: 60px;"><?php _e( 'Height:', 'simplemap' ); ?> </label>
+															       style="display: inline-block; width: 60px;"><?php _e( 'Height:', 'SimpleMap' ); ?> </label>
 															<input type="text" name="map_height" id="map_height"
 															       size="13"
 															       value="<?php echo esc_attr( $map_height ); ?>"/><br/>
 															<small>
-																<em><?php printf( __( 'Enter a numeric value with CSS units, such as %s or %s.', 'simplemap' ), '</em><code>100%</code><em>', '</em><code>500px</code><em>' ); ?></em>
+																<em><?php printf( __( 'Enter a numeric value with CSS units, such as %s or %s.', 'SimpleMap' ), '</em><code>100%</code><em>', '</em><code>500px</code><em>' ); ?></em>
 															</small>
 														</td>
 													</tr>
 
 													<tr valign="top">
 														<td><label
-																for="map_type"><?php _e( 'Default Map Type', 'simplemap' ); ?></label>
+																for="map_type"><?php _e( 'Default Map Type', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 134 characters

<!-- Comment by [SideCI](https://sideci.com) -->